android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/SettingsFragment.java

```diff
@@ -16,10 +16,20 @@
 
 package org.physical_web.physicalweb;
 
+import android.content.SharedPreferences;
```

Comment:
Yes, sorry forgot to run the check. Doing so now with let you know when it is good.